### PR TITLE
Initial Migrations for DB

### PR DESCRIPTION
**Pull request by CalvinRodo**
These scripts should create a database. 

## To Test 

Start a database
Change `AUTO_MIGRATE_MODE` in the `.env` file to "safe"
Run `npm run db:migrate` to run the migrations
Run `npm run dev` to start the project
Run `npm run cypress:open` to open cypress then run the tests.
Make sure all tests pass

Also just play around make sure nothing breaks.

### CODE REVIEW

**Comment by jeffmaher** [comment]
Taking a look now

**Comment by CalvinRodo** [comment]
@jeffmaher I fixed the issues and added information to the Readme.

File: database/migrations/20200222035426-create-users.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
+  down: (queryInterface) => {
+    return queryInterface.dropTable('Users');
+  }
+};
```
Comment (dsamojlenko) [inline database/migrations/20200222035426-create-users.js]: Missing the isAdmin boolean column

Comment (CalvinRodo) [reply database/migrations/20200222035426-create-users.js]: Fixed and tested

**Pull request merged**